docs/data/data-grid/export/CsvGetRowsToExportRowGrouping.js

```diff
@@ -71,7 +71,6 @@ export default function CsvGetRowsToExportRowGrouping() {
             model: ['commodity'],
           },
         }}
-        experimentalFeatures={{ rowGrouping: true }}
```

Comment:
Are we sure it's soon enough to remove the experimental flag on this feature? Did we have enough product feedback to be confident it won't require a BC? At a minimum, I would encourage we don't bundle in the same PR a new feature with a change of another feature going from experimental to stable.

Reply:
For me we only added the experiment feature due to the plan.
The feature itself is marked as "stable" on the doc

![image](https://user-images.githubusercontent.com/3309670/161698444-a87eb293-0926-4111-b0c4-a64787df87de.png)

If the Row Grouping was intended for pro, I would never have been under an experimental flag in the first place.
So removing the experimental flag seems logical to me since we are removing the only reason why it exists.

Reply:
Thanks for the clarification. Ok, so it's only something we talk as an option in https://www.notion.so/mui-org/Version-Sync-Strategy-between-X-and-Core-f877ece7d0d04c339a2b929b96164211#449ab8b4db6d4c228d055a236932e93c.